Formula/clojure-lsp.rb

```diff
@@ -30,14 +30,17 @@ def install
   test do
     require "Open3"
 
-    stdin, stdout, _, wait_thr = Open3.popen3("#{bin}/clojure-lsp")
+    system "file", "#{bin}/clojure-lsp"
```

Comment:
This line is no longer needed.

Also, you're using `assert_match`, so it doesn't matter that it's printing more than you need (you only need to match a substring).

I think you need to set `JAVA_HOME` properly. Try
```ruby
ENV["JAVA_HOME"] = Formula["openjdk@11"].opt_prefix
```

Reply:
> Also, you're using assert_match, so it doesn't matter that it's printing more than you need (you only need to match a substring).


`stdout.gets` will consume per line, so in the current approach, we do need to know how many times it will be called. But surely there's a better way of doing this.

Anyway, I'm adding the ENV and removing the unnecessary line.